sdks/python/src/opik/api_objects/dataset/dataset.py

```diff
@@ -205,6 +205,25 @@ def insert_from_json(
 
         self.insert(new_items)
 
+    def read_jsonl_from_file(
+        self,
+        file_path: str,
+        keys_mapping: Optional[Dict[str, str]] = None,
+        ignore_keys: Optional[List[str]] = None,
+    ) -> None:
+        """
+        Read JSONL from a file and insert it into the dataset.
+
+        Args:
+            file_path: Path to the JSONL file
+            keys_mapping: dictionary that maps json keys to item fields names
+                Example: {'Expected output': 'expected_output'}
+            ignore_keys: if your json dicts contain keys that are not needed for DatasetItem
+                construction - pass them as ignore_keys argument
+        """
+        new_items = converters.from_jsonl_file(file_path)
+        self.insert_from_json(json.dumps(new_items), keys_mapping, ignore_keys)
```

Comment:
after `converters.from_jsonl_file` starts returning `List[DatasetItem]` instead of `List[Dict[str, Any]]`, this will turn into 
`self.insert(new_items).